lib/resizers/resizer_stream.js

```diff
@@ -37,7 +37,7 @@ ResizerStream.prototype.baseParameters = function() {
     output = this.options.convertTo + ':-';
   }
 
-  return ['-quality', '91', '+profile', '*', '-auto-orient', '-strip', output];
+  return ['-quality', this.options.quality ? this.options.quality : '91', '+profile', '*', '-auto-orient', '-strip', output];
```

Comment:
And maybe it's a good idea to put the default quality in a variable... something like 
`var DEFAULT_QUALITY='91'`